### DISCUSSION TITLE
Rental NFTs

### DISCUSSION THREAD

**Discussion by peshwar9** [capsule-corp-ternoa/ternoa-proposals#40] [Announcements]
Starting a thread on Rental NFTs.

What would be their utility? Who will use it? What are the possible use cases? What specific onchain functionality would you like to see?

Look forward to seeing your thoughts here.

**Comment by markopoloparadox** [comment]
From a gaming perspective if we have a feature like rent then players could rent their equipment to other players for a fee which could be useful. This is not only limited to the gaming industry, this could work with any NFT that is linked to a real life object, like digital car key, digital house keys, etc...

Could this also be an opportunity to have something like subscriptions? I think that subscriptions are the same thing as renting but the contract is renewed automatically every month. This would allow games to sell subscription based features.

**Reply by peshwar9** [reply]
Yeah, subscriptions are a good idea. But do you think they can be used beyond just games?

**Reply by markopoloparadox** [reply]
It definitely goes beyond gaming. Our subscriptions model can work just like gym/netflix subs works now.

**Reply by Herrlalwani** [reply]
subscriptions could also be looked into other industries like automobile leasing, education, real estate/furniture renting, skincare/nutrition, etc.

**Comment by jeemclr** [comment]
# The future of subscription: "Rental NFTs"

Really interesting ideas from [cantino.eth](https://twitter.com/chriscantino) : 
[https://twitter.com/chriscantino/status/1557109060431794176?s=21&t=Z7fkSIE4NYqeB1nMKDuqfQ](https://twitter.com/chriscantino/status/1557109060431794176?s=21&t=Z7fkSIE4NYqeB1nMKDuqfQ)

### Why subscription NFTs?

Web2 subscriptions are rented, not owned. You do not own your Spotify membership. You can not sell it. And due to Apple’s 30% commission, you are also paying “platform tax.” But this is just first inning thinking. Here’s what they’re really capable of.

## Feature Unlocks and Exclusives

- Tesla NFTs that unlock access to premium features.
- HBO NFTs that let you watch new movies before release.
- Food Network NFTs that serve up interactive cooking lessons.
- By attaching scarce perks to NFT-based subscriptions, their value increases.

## IP & Royalties

- Netflix PFPs that puts subscriber's characters “in the show.”
- Spotify NFTs that stake to royalties from your most listened-to artists.
- Crunchyroll NFTs that grant commercial rights to anime IP.
- NFTs enable each subscription to have unique market value.

## Airdrops

- Disney+ subscriptions that airdrop NFTs of Marvel artwork and Star Wars PFPs.
- Tidal NFTs that airdrop fractional ownership of music vids.
- Coinbase NFTs with access to premium market data and NFT airdrops.
- Airdrops are great for engaging and keeping subscribers happy.

## Composability

- Web3’s killer feature is interoperability.
- Imagine your Criterion Collection NFT being forked to unlock access to communities of film enthusiasts.

Or, a new coffeeshop trying to attract business:

- "One free drink for holders of Starbucks loyalty NFTs."

## Loyalty Perks

- A StockX subscription NFT that guarantees drop access to their most loyal customers, and continuously updates the NFT’s metadata to reflect its value.
- Reward top customers, but allow them to sell their NFT on the open market if they so choose.

## The “Team Staking” Subscription

- A premium Netflix Subscription that only activates when several holders stake together. This could enable access IRL showings or celeb meet-and-greets.
- It also encourages loyalty as members remain staked so as to not upset their co-stakers.

## Performance-Based Subscriptions

A Fortnite PFP that doubles as an NFT Subscription where all members get access to exclusive skins and maps, but top players unlock rare traits and exclusive events.

The better the player, the more valuable their NFT.

## Evolving Passes

- Apple Health NFTs that update to reflect your exercise routine.
- Calm NFTs that increase in rarity the more you meditate.
- Peloton NFTs that grow in value the more you ride.

Because NFTs aren’t static, brands can tell stories and add layers of value.

**Comment by Herrlalwani** [comment]
**Purpose of NFT Rental is maybe someone can't afford to buy an NFT that they need, so they're looking for ways to rent it?**

Imagine you own an NFT that has utility. You can either use that utility yourself or you can rent NFTs out to someone else for a fee.
There are various reasons why someone may want to rent NFTs. Some what to flex an expensive NFT as their profile pic. Others want to use them in a game because it may help them level up faster. And some NFTs come with exclusive access privileges to concerts or content.

**Renting out NFTs may eventually become a reasonably lucrative business model with attractive passive income streams.**

**Comment by Victor-Salomon** [comment]
Hi friends, 

### Intro : 

Focused into Rental NFT integration, I started to look a bit on what is done on the market. Quickly appear the concept around the **Collateralized** or **Collateral-less/Collateral-free renting** where, the way to provide the _**rental NFT feature includes more or less protection for the NFT Owner.**_ 

### Concept : 

In Collateralized renting case : 
To protect the renter, the borrower who want to rent the NFT must deposit a collateral that worth more than the NFT to safeguard the renter’s interests. (or likely exceeding the value of the NFT.) The value of each NFT is likely verified using price oracles. In addition, the rental fee he must pay. The NFT will be returned to the original owner after the contract expires, and the borrower will receive their collateral back.

In Collateral-less/free renting case : 
The borrower never receives the original NFT, unlike collateral renting and a wrapped version of the original NFT is minted. This wrapped NFT carries all of the same characteristics and attributes of the original asset and is, of course, backed by that asset. The wrapped NFT is burned once the contract ends. In this situation, neither the renter nor the owner is required to put up any collateral. The financial risk is reduced for both parties as a result.

### More in detail : 

I'm going to look a bit into ERC-standards as I'm not fully confortable for now with the good version to share. It seems that Rental when from ERC-809 to ERC-1201 to now ERC-4907.(https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4907.md). 
I'll also look at how Unique and reNFT are dealing with their Rental NFT.

### Reflexion/Solution ?

**Even if the exact notion of _collateral vs collateral-less/free_ is or is not usefull for us, I think a general concept of collateral should be still interesting. Adding a collateral/caution could be required to cover some usecases (I agree that most of them are real-world related... but let's see far away in the future of Web3 addoption). We already talked about automobile leasing, airbnb, real estate/furniture(...).**

**Reply by peshwar9** [reply]
@Victor-Salomon I think collaterals are more useful in case of DeFi loans. In case of NFT renting, we are not really transferring the NFT to the rentee, so the NFT is secure for the renter anyways. What value does collateral add here?

**Reply by Victor-Salomon** [reply]
@peshwar9 what about a contract that allow you to lease a car (@ondrums777 talked about it), and you damage it. What would happen, because usually contracts like this include a collateral? Or what about some use case like renting an appartment ? what's happen usually, is that landlord is requiring a month of rent.

see exemple of : https://www.renft.io/

**Reply by peshwar9** [reply]
@Victor-Salomon oh, I wasn't thinking irl assets issued as nfts, was just focusing on digital assets. In real world assets (RWA) collateral makes sense. But rn we don't have the legal framework integrated with NFTs to do the kind of examples you provided (such as renting airbnb or renting a car).

**Reply by Victor-Salomon** [reply]
@peshwar9 Ok understood. I can't wait for more NFT usecases  in our daily life 🚀

**Reply by jeemclr** [reply]
The closest EIP than your looks this one : https://eips.ethereum.org/EIPS/eip-2615
